src/components/ModalPayment/ModalAmount/LanternForm.tsx

```diff
@@ -0,0 +1,278 @@
+      },
+    });
+  };
+
+  const handleAddress = (e, bool) => {
+    e.preventDefault();
+    setShowAddress(bool);
+
+    if (!bool) {
+      dispatch({
+        type: ModalPaymentConstants.CLEAR_ADDRESS,
+        payload: null,
+      });
+    }
+  };
+
+  return (
+    <AmountContainer>
+      <Subheader>{t(subHeader)}</Subheader>
+      <label>{t(subText)}</label>
+      <br />
+      <br />
+      <label>{t('paymentProcessing.amount.personalize')}</label>
+      <br />
+      <br />
+      <SingleRowFormat>
+        <RowFormat width="38%">
+          <LabelText htmlFor="first_name">
+            {t('paymentProcessing.amount.labels.first_name')}
+          </LabelText>
+          <InputText
+            name="firstName"
+            type="text"
+            className="modalInput--input"
+            onChange={handleOnChange}
+            placeholder={t('paymentProcessing.amount.place_holder.first_name')}
+            value={lucData.firstName}
+          />
+        </RowFormat>
+        <RowFormat width="20%" mobileWidth="30%">
+          <LabelText htmlFor="middle_initial">
+            {t('paymentProcessing.amount.labels.middle_initial')}
+          </LabelText>
+          <InputText
+            name="middleInitial"
+            type="text"
+            className="modalInput--input"
+            onChange={handleOnChange}
+            placeholder={t(
+              'paymentProcessing.amount.place_holder.middle_initial'
+            )}
+            value={lucData.middleInitial}
+          />
+        </RowFormat>
+        <RowFormat width="38%">
+          <LabelText htmlFor="last_name">
+            {t('paymentProcessing.amount.labels.last_name')}
+          </LabelText>
+          <InputText
+            name="lastName"
+            type="text"
+            className="modalInput--input"
+            onChange={handleOnChange}
+            placeholder={t('paymentProcessing.amount.place_holder.last_name')}
+            value={lucData.lastName}
+          />
+        </RowFormat>
+      </SingleRowFormat>
+
+      {amount >= LIGHT_UP_CHINATOWN_TIER_2_LIMIT && (
+        <>
+          <label>
+            <span>
+              {' '}
+              {t(
+                'paymentProcessing.amount.light_up_chinatown_address.disclaimer_1'
+              )}
+            </span>
+            <b>
+              {' '}
+              {t('paymentProcessing.amount.light_up_chinatown_address.address')}
+            </b>
+            <span>
+              {' '}
+              {t(
+                'paymentProcessing.amount.light_up_chinatown_address.disclaimer_2'
+              )}
+            </span>
+          </label>
+          <br />
+          <br />
+
+          <TextTrigger onClick={(e) => handleAddress(e, true)}>
+            {' '}
+            {t(
+              'paymentProcessing.amount.light_up_chinatown_address.address_button'
+            )}
+          </TextTrigger>
+          <br />
+          <br />
+
+          {showAddress && (
```

Comment:
```suggestion
          {(showAddress || lucData.address !== '') && (
```
If the user goes back to edit the address, the part with their address details will show up

Reply:
bump